content/en/real_user_monitoring/browser/advanced_configuration.md

```diff
@@ -1035,3 +1185,6 @@ However, this feature comes with some **limitations**:
 [17]: https://github.com/DataDog/browser-sdk/blob/main/CHANGELOG.md#v4130
 [18]: /data_security/real_user_monitoring/#browser-rum-use-of-cookies
 [19]: https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage
+[20]: /real_user_monitoring/browser/advanced_configuration/?tab=npm#override-default-rum-view-names
```

Comment:
It looks like this link (`[20]`) didn't get added anywhere. Was it meant to be added somewhere?

Reply:
I removed this link and updated the other one.